src/themes/dojo/comboBox.m.css

```diff
@@ -1,55 +1,103 @@
 @import './variables.css';
 
 .root {
-	position: relative;
-	display: inline-block;
 	box-sizing: border-box;
-	width: var(--form-field-width);
+	color: var(--color-text-primary);
+	display: inline-block;
+	font-size: var(--font-size-base);
+	line-height: var(--line-height-base);
+	position: relative;
+	width: 100%;
 }
 
 .root * {
 	box-sizing: border-box;
 }
 
-.controls {
-	position: relative;
+.root input {
+	line-height: var(--line-height-base);
+	min-width: 200px;
+	padding: var(--spacing-regular) calc(var(--spacing-regular) * 6) var(--spacing-regular) var(--spacing-regular);
+	width: 100%;
+}
+
+.trigger {
+	appearance: none;
+	background-color: transparent;
+	border-bottom: 0;
+	border-left: var(--border-width) solid var(--color-border);
+	border-radius: 0;
+	border-right: 0;
+	border-top: 0;
+	bottom: var(--border-width);
+	color: var(--color-text-faded);
+	cursor: pointer;
+	display: block;
+	font-size: inherit;
+	font-size: var(--font-size-base);
+	line-height: var(--line-height-base);
+	outline: 0;
+	padding: var(--spacing-regular) calc(var(--spacing-regular) + 4);
+	position: absolute;
+	right: 0;
+	top: var(--border-width);
+	z-index: 1;
 }
 
-.input {
+.dropdown {
+	background-color: var(--color-background);
+	border: var(--border-width) solid var(--color-border);
+	box-shadow: var(--box-shadow-dimensions-small) var(--color-box-shadow);
+	opacity: 0;
+	position: absolute;
+	transform-origin: 0 0;
+	transform: rotateX(90deg);
+	transition: all var(--transition-duration) var(--transition-easing);
 	width: 100%;
-	padding-right: 2em;
+	z-index: 2;
 }
 
-.input::-ms-clear {
-	display: none;
+.open .dropdown {
+	opacity: 1;
+	transform: rotateX(0);
 }
 
-.clearable {
-	padding-right: 4em;
+.option {
+	background-color: var(--color-background);
+	border: var(--border-width) solid transparent;
+	cursor: pointer;
+	line-height: var(--line-height-base);
+	padding: var(--spacing-regular);
+	transition: background var(--transition-duration) var(--transition-easing), border var(--transition-duration) var(--transition-easing);
 }
 
-.results {
-	position: absolute;
-	width: 100%;
-	max-height: var(--select-menu-height);
-	overflow-y: auto;
-	background: var(--dojo-white);
-	border: var(--border-thin) solid var(--dojo-shade-gray);
-	z-index: 1;
+.selected {
+	background-color: var(--color-background-faded);
 }
 
-.result {
-	cursor: pointer;
-	padding: var(--input-padding);
+.invalid .trigger {
+	border-left-color: color(var(--color-error) saturation(-9%) lightness(+37%));
```

Comment:
This color (and the success variation) is getting used in more places than I thought. Do you think it's worth thinking of another variable name?